external/CMakeLists.txt

```diff
@@ -61,6 +61,8 @@ list(APPEND EXTERNAL_INCLUDE_DIRS ${LIB_STATIC_APRUTIL1_INC_DIR})
 
 include(YamlLib)
 set(LIB_STATIC_YAML_LOC ${LIB_STATIC_YAML_LOC} PARENT_SCOPE)
+set(LIB_STATIC_YAML_INC_DIR ${LIB_STATIC_YAML_INC_DIR} PARENT_SCOPE)
+list(APPEND EXTERNAL_INCLUDE_DIRS ${LIB_STATIC_YAML_INC_DIR})
```

Comment:
What's different about the ARM build from other builds that would require adding `LIB_STATIC_YAML_INC_DIR` to `EXTERNAL_INCLUDE_DIRS`. Is this really necessary?